lib/yoti/protobuf/main.rb

```diff
@@ -18,6 +19,10 @@ class << self
       CT_JSON = :JSON # json_string
       CT_INT = :INT # integer
 
+      def logger
```

Comment:
I was thinking if we're going to use this logger in other places, it might be good to define it in a common class. But I guess we can just do that whenever we implement needing it in another place

Reply:
I've moved this into a separate class in ecb14ad